src/router.ts

```diff
@@ -1,5 +1,5 @@
 import Vue from 'vue';
-import VueRouter from 'vue-router';
+import VueRouter, { Route } from 'vue-router';
```

Comment:
Our TypeScript linter(s) threw a bogus "defined but never used (no-unused-vars)" error on the Route interface despite its use below. "You had ONE job..." After following the issue-report bounces across three different GitHub projects, I tried the following upgrades to no avail:

```
-    "@vue/eslint-config-typescript": "^3.0.5",
+    "@vue/eslint-config-typescript": "^3.1.1",
+    "eslint-plugin-typescript": "^0.13.0",
+    "typescript-eslint-parser": "^21.0.1",
```